src/content/guides/typescript.mdx

```diff
@@ -94,6 +95,45 @@ module.exports = {
 
 This will direct webpack to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
 
+Regarding webpack configuration, instead of using Javascript for `webpack.config.js`, we could use the Typescript version `webpack.config.ts`. This is possible by installing some packages below.
+
+```bash
+npm install --save-dev ts-node @types/webpack
```

Comment:
No need to install `@types/webpack`, webpack v5 has built-in types

Reply:
That's right! I'll remove this part. I was putting it here because I ran this guide with webpack4, but I guess this always targets the latest version which is webpack5.  👍